.github/renovate.json5

```diff
@@ -43,6 +43,14 @@
       platformAutomerge: true
     },
 
+    // Dockerfile for google-cloud-cli, basically no minor/patch version, daily major version updates
```

Comment:
The comment seems off.

Reply:
Ehm - yea. I initially only saw the gcloud bump and then jaeger jumped in :facepalm: